bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/ReadEntryProcessor.java

```diff
@@ -129,6 +129,10 @@ protected void processPacket() {
         } catch (BookieException e) {
             LOG.error("Unauthorized access to ledger " + read.getLedgerId(), e);
             errorCode = BookieProtocol.EUA;
+        } catch (Throwable t) {
+            LOG.error("Unexpected exception reading at {}:{} : {}",
+                    new Object[] { read.getLedgerId(), read.getEntryId(), t.getMessage(), t });
```

Comment:
I think in current version of slf4j, you don't need to use `new Object[] {}` any more.